src/common/enums/index.ts

```diff
@@ -9,4 +9,11 @@ enum ProductStatus {
   BOTH = 'both',
 }
 
-export { Gender, ProductStatus };
+enum ProductActivityStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+  PUBLISHED = 'published',
+  REJECTED = 'rejected',
+}
```

Comment:
I think that we decided to make only three statuses. Or we will change it later?

Reply:
I think it will be better change while adding products. and I will know for sure. but I can change I need for this task only active status